Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM sonatype/nexus3:3.1.0
+MAINTAINER Dwolla Dev <dev+docker-nexus3-crowd@dwolla.com>
+LABEL org.label-schema.vcs-url="https://github.com/Dwolla/docker-nexus3-crowd"
+
+USER root
```

Comment:
Do we actually need to run nexus as `root`, or can we switch to a different user after things are installed?